heft-plugins/heft-typescript-plugin/src/configureProgramForMultiEmit.ts

```diff
@@ -51,7 +51,9 @@ export function configureProgramForMultiEmit(
     // Attach the originals to the Program instance to avoid modifying the same Program twice.
     // Don't use WeakMap because this Program could theoretically get a { ... } applied to it.
     [INNER_GET_COMPILER_OPTIONS_SYMBOL]?: TTypescript.Program['getCompilerOptions'];
-    [INNER_EMIT_SYMBOL]?: TTypescript.Program['emit'];
+    [INNER_EMIT_SYMBOL]?: // https://github.com/microsoft/TypeScript/blob/88cb76d314a93937ce8d9543114ccbad993be6d1/src/compiler/program.ts#L2697-L2698
+    // There is a "forceDtsEmit" parameter that is not on the published types.
+    (...args: [...Parameters<TTypescript.Program['emit']>, boolean | undefined]) => TTypescript.EmitResult;
```

Comment:
Could this be updated to include the optional param and avoid the tsignores?

Reply:
Done.